### PR TITLE
Remove JITSERVER_TODO macro

### PR DESCRIPTION
**Pull request by harryyu1994**
Signed-off-by: Harry Yu <harryyu1994@gmail.com>

### CODE REVIEW

**Comment by harryyu1994** [comment]
@mpirvu This one is ready for review.

File: runtime/compiler/runtime/CMakeLists.txt [hunk 1]
```diff
@@ -62,6 +62,7 @@ j9jit_files(
 
 if(JITSERVER_SUPPORT)
 	j9jit_files(
+	        rumtime/CompileService.cpp
```
Comment (mpirvu) [inline runtime/compiler/runtime/CMakeLists.txt]: typo: "rumtime"

File: runtime/compiler/runtime/CMakeLists.txt [hunk 2]
```diff
@@ -62,6 +62,7 @@ j9jit_files(
 
 if(JITSERVER_SUPPORT)
 	j9jit_files(
+                runtime/CompileService.cpp
```
Comment (mpirvu) [inline runtime/compiler/runtime/CMakeLists.txt]: Could you please align this line with the rest?

Comment (harryyu1994) [reply runtime/compiler/runtime/CMakeLists.txt]: having a hard time aligning, one moment

File: runtime/compiler/runtime/CMakeLists.txt [hunk 3]
```diff
@@ -62,6 +62,7 @@ j9jit_files(
 
 if(JITSERVER_SUPPORT)
 	j9jit_files(
+	    runtime/CompileService.cpp
```
Comment (mpirvu) [inline runtime/compiler/runtime/CMakeLists.txt]: Now there are to few spaces :-)

**Comment by mpirvu** [comment]
Jenkins test sanity all jdk8,jdk11

**Comment by mpirvu** [comment]
All tests have passed except win64 which failed due to infra.
```
ERROR: Error fetching remote repo 'origin'
hudson.plugins.git.GitException: Failed to fetch from https://github.com/AdoptOpenJDK/openjdk-tests.git
```
PR is good to be merged

**Pull request merged**